Source/Core/Common/Hash.h

```diff
@@ -15,4 +15,5 @@ u32 HashAdler32(const u8* data, size_t len);         // Fairly accurate, slightl
 u32 HashEctor(const u8* ptr, size_t length);         // JUNK. DO NOT USE FOR NEW THINGS
 u64 GetHash64(const u8* src, u32 len, u32 samples);
 void SetHash64Function();
+std::string MakeGameID(std::string file_name);
```

Comment:
I'm unsure where this function should be added, but this shouldn't be here in Common/Hash.h.

Maybe make this a SConfig static member function for now, since that's where we have the "current running game metadata" logic...